packages/js/components/src/experimental-select-control/stories/index.tsx

```diff
@@ -176,25 +176,35 @@ export const CustomRender: React.FC = () => {
 		setSelected( [ ...selected, item ] );
 	};
 
+	const getFilteredItems = ( allItems: ItemType[], inputValue: string ) => {
```

Comment:
I think this needs to be updated to `DefaultItemType`.

```suggestion
	const getFilteredItems = ( allItems: DefaultItemType[], inputValue: string ) => {
```

Reply:
Good catch